nexus/value/src/lib.rs

```diff
@@ -164,7 +165,7 @@ impl Value {
                 } else if let Some(f) = n.as_f64() {
                     Value::Double(f)
                 } else {
-                    Value::Numeric(n.to_string())
+                    Value::Numeric(n.to_string().as_str().parse::<Decimal>().unwrap())
```

Comment:
rather than `unwrap` send the error upwards if possible.

Reply:
Would require refactoring return types and values of this function and functions where this is called.. don't see an easy way right now